diy_asylum_frontend/src/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
```

Comment:
`import { BrowserRouter as Router, Route, Switch } from "react-router-dom";`

(related to comment below)